Source/ablastr/fields/PoissonSolver.H

```diff
@@ -11,8 +11,12 @@
 #include <ablastr/utils/Communication.H>
 #include <ablastr/utils/TextMsg.H>
 #include <ablastr/warn_manager/WarnManager.H>
+#include <ablastr/math/fft/AnyFFT.H>
 #include <ablastr/fields/Interpolate.H>
 
+#if defined(WARPX_USE_PSATD)  && defined(WARPX_DIM_3D)
```

Comment:
Macros are ok. We  have yet to introduce separate FFT macros for ABLASTR (will do another time, already not done for other FFT functions in ABLASTR).